Pipfile

```diff
@@ -21,6 +21,7 @@ gunicorn = {version = "*", markers="sys_platform == 'linux'"}
 parse = "*"
 importlib-metadata = {version = "*", markers="python_version < '3.8'"}
 colorama= {version = "*", markers="sys_platform == 'win32'"}
+myst-parser = "*"
 
```

Comment:
We also need to update requirements.txt in docs/. 
Otherwise the build in Read the docs will fail.

That is ofcourse only a middle step.
What I would really like is to have `tests`  and `docs` categories in Pipfile, and then we can only install `docs` category in the build.

Reply:
Also - there is a missing dependency:
```
myst-parser = {extras = ["linkify"], version = "*"}
```